src/components/views/rooms/EventTile.js

```diff
@@ -630,8 +636,12 @@ module.exports = withMatrixClient(React.createClass({
             onFocusChange={this.onActionBarFocusChange}
         /> : undefined;
 
-        const timestamp = this.props.mxEvent.getTs() ?
-            <MessageTimestamp showTwelveHour={this.props.isTwelveHour} ts={this.props.mxEvent.getTs()} /> : null;
+        const timestamp = this.props.mxEvent.getTs()
+            ? <MessageTimestamp
+                showTwelveHour={this.props.isTwelveHour}
+                ts={this.props.mxEvent.getTs()}
+                ariaHidden={muteScreenReader}
```

Comment:
Hmm, if it's muted when sending or sending is complete, when will it be read...?

Reply:
The life of an EventTile is 3 parts, where the third is when things are read out. The first stage is the sending state, which is easy to filter. When the event switches to sent, the tile unmounts and a new one is put in place with a null sending state. That state is replaced shortly after with a property update, which triggers a re-render where the screen reader picks up the text to read (the div is replaced).

Reply:
Just for my own understanding, what does "that state is replaced shortly after with a property update" mean? Which property updates...?

Reply:
The mxEvent changes in the component's properties, which leads to an eventSendState change of null->sent

Reply:
Huh, I thought `sent` was first (while waiting for remote echo), then `null` was the last and final status...

Reply:
apparently it's just not my day - you're right. Will open another PR to flip it around.